src/sentry/static/sentry/app/plugins/sessionstack/contexts/sessionstack.jsx

```diff
@@ -1,55 +1,51 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import PropTypes from "prop-types";
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
 
 const ASPECT_RATIO = 16 / 9;
 
 class SessionStackContextType extends React.Component {
-  static propTypes() {
-    return {
-      alias: PropTypes.string.isRequired,
-      data: PropTypes.object.isRequired
-    };
-  }
+  propTypes = {
+    data: PropTypes.object.isRequired,
+  };
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      showIframe: false
-    };
-  }
+  state = {
+    showIframe: false,
+  };
 
   componentDidMount() {
+    // eslint-disable-next-line react/no-find-dom-node
     this.parentNode = ReactDOM.findDOMNode(this).parentNode;
-    window.addEventListener("resize", () => this.setIframeSize(), false);
+    window.addEventListener('resize', () => this.setIframeSize(), false);
```

Comment:
I realize this is just old code, but we should probably debounce this @scefali

Reply:
@billyvg Makes sense. I would prefer to tackle it in a different PR though, however.